deploy/kyma/charts/sustainable-saas/charts/susaas-router/templates/network-policy.yaml

```diff
@@ -15,11 +15,11 @@ spec:
             app: istio-ingressgateway
         namespaceSelector:
           matchLabels:
-            name: istio-system
+            kubernetes.io/metadata.name: istio-system
       - podSelector:
           matchLabels:
             app.kubernetes.io/name: oathkeeper
             app.kubernetes.io/instance: ory
         namespaceSelector:
           matchLabels:
-            name: kyma-system
\ No newline at end of file
+            name: kyma-system
```

Comment:
Shouldn't this label be changed as well to `kubernetes.io/metadata.name: kyma-system` ?